### PR TITLE
Image refresh for fedora-30

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for fedora-30
 * [x] image-refresh fedora-30

### CODE REVIEW

**Comment by marusak** [comment]
There are two [failed tests](https://209.132.184.41:8493/logs/pull-11597-20190415-074141-f1504a14-verify-fedora-30/log.html) and interestingly enough exactly the same two (with the same log) [failed](https://209.132.184.41:8493/logs/pull-11596-20190415-072424-8cef4f5f-verify-fedora-30/log.htm) in duplicated PR #11596

**Comment by marusak** [comment]
testCPUSecurityMitigations (check_system_info.TestSystemInfo) and  testStoragePoolsCreate (check_machines_dbus.TestMachinesDBus) failed both 3 times. Needs work

**Comment by marusak** [comment]
`testStoragePoolsCreate` fails because selinux prevents `iscsid` to load some modules, I think this will need BZ and naughty.

```
systemd[1]: Started Open-iSCSI.
audit[7809]: AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.softdep" dev="dm-0" ino=8480266 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
kernel: audit: type=1400 audit(1555325952.565:469): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.softdep" dev="dm-0" ino=8480266 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
libvirtd[4448]: internal error: Child process (iscsiadm --mode node --portal 127.0.0.1:3260,1 --targetname iqn.2019-09.cockpit.lan --login) unexpected exit status 12: iscsiadm: Could not login to [iface: default, target: iqn.2019-09.cockpit.lan, portal: 127.0.0.1,3260].
                                                      iscsiadm: initiator reported error (12 - iSCSI driver not found. Please make sure it is loaded, and retry the operation)
                                                      iscsiadm: Could not log into all portals
audit[7809]: AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
kernel: audit: type=1400 audit(1555325952.572:470): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.builtin.bin" dev="dm-0" ino=8480269 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
kernel: audit: type=1400 audit(1555325952.572:471): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.builtin.bin" dev="dm-0" ino=8480269 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
audit[7809]: AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
kernel: audit: type=1400 audit(1555325952.573:472): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
audit[7809]: AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
kernel: audit: type=1400 audit(1555325952.573:473): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
iscsid[7808]: Could not insert module tcp. Kmod error -2
```

**Comment by KKoukiou** [comment]
> `testStoragePoolsCreate` fails because selinux prevents `iscsid` to load some modules, I think this will need BZ and naughty.
> 
> ```
> systemd[1]: Started Open-iSCSI.
> audit[7809]: AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.softdep" dev="dm-0" ino=8480266 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> kernel: audit: type=1400 audit(1555325952.565:469): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.softdep" dev="dm-0" ino=8480266 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> libvirtd[4448]: internal error: Child process (iscsiadm --mode node --portal 127.0.0.1:3260,1 --targetname iqn.2019-09.cockpit.lan --login) unexpected exit status 12: iscsiadm: Could not login to [iface: default, target: iqn.2019-09.cockpit.lan, portal: 127.0.0.1,3260].
>                                                       iscsiadm: initiator reported error (12 - iSCSI driver not found. Please make sure it is loaded, and retry the operation)
>                                                       iscsiadm: Could not log into all portals
> audit[7809]: AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> kernel: audit: type=1400 audit(1555325952.572:470): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.builtin.bin" dev="dm-0" ino=8480269 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> kernel: audit: type=1400 audit(1555325952.572:471): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.builtin.bin" dev="dm-0" ino=8480269 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> audit[7809]: AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> kernel: audit: type=1400 audit(1555325952.573:472): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> audit[7809]: AVC avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> kernel: audit: type=1400 audit(1555325952.573:473): avc:  denied  { read } for  pid=7809 comm="iscsid" name="modules.dep.bin" dev="dm-0" ino=8668433 scontext=system_u:system_r:iscsid_t:s0 tcontext=system_u:object_r:modules_dep_t:s0 tclass=file permissive=0
> iscsid[7808]: Could not insert module tcp. Kmod error -2
> ```

Created downstream issue, naughty and knownissue for that. Thanks :)

**Comment by martinpitt** [comment]
The CPUSecurityMitigations test fails on current fedora-30 as well, so that's not a regression

**Comment by KKoukiou** [comment]
> The CPUSecurityMitigations test fails on current fedora-30 as well, so that's not a regression

Also[ rhel-8-0](https://209.132.184.41:8493/logs/pull-11574-20190416-054141-5e2a6061-verify-rhel-8-0/log.html) and [rhel-8-1 ](https://209.132.184.41:8493/logs/pull-11574-20190416-054141-5e2a6061-verify-rhel-8-1/log.html)

File: bots/naughty/fedora-30/11614-selinux-denying-iscsi_tcp-module-load [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Traceback (most recent call last):
+  File "/build/cockpit/bots/../test/verify/check-machines-dbus", line *, in testStoragePoolsCreate
```
Comment (marusak) [inline bots/naughty/fedora-30/11614-selinux-denying-iscsi_tcp-module-load]: The file path can be specified as `test/verify/check-machines-dbus`.  Also maybe the naughty would be a bit safer if you would specify that the dialog also fails to close, but I think it is just fine :)
Now fails only testCPUSecurityMitigations which is regression outside of this PR.

**Pull request merged**